src/Source/Framework/Bio.Core/IO/BAM/BAMParser.cs

```diff
@@ -13,13 +13,22 @@
 
 namespace Bio.IO.BAM
 {
+	/// <summary>
+	/// Default BAM parser with template parameter set to SAMAlignedSequence for general use inside the .NET Bio libraries
+	/// </summary>
+	public partial class BAMParser : BAMParser<SAMAlignedSequence>
```

Comment:
@markjulmar What's your feeling about the repeated use of the "BAMParser" name here?